From 6c28ca8d95f7f485e7c59020f1c6e41245e3e312 Mon Sep 17 00:00:00 2001
From: Keerthana K <keerthanak@vmware.com>
Date: Wed, 16 Nov 2022 09:06:07 +0000
Subject: [PATCH] stalld: change default config_granularity value to 2s

Changing the default config_granularity to 2 seconds which is
configure for Telco customers. If failed to configure
config_granularity through config file, then default value from
code takes precedence.

Signed-off-by: Keerthana K <keerthanak@vmware.com>
---
 src/stalld.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/stalld.c b/src/stalld.c
index dccf0a8..30d5641 100644
--- a/src/stalld.c
+++ b/src/stalld.c
@@ -76,7 +76,7 @@ unsigned long config_force_fifo = 0;
 long config_starving_threshold = 20;
 long config_boost_duration = 3;
 long config_aggressive = 0;
-long config_granularity = 5;
+long config_granularity = 2;
 
 /*
  * XXX: Make it a cpu mask, lazy Daniel!
-- 
2.28.0

diff --git a/src/stalld.c b/src/stalld.c
index a71e7dd..62d2b01 100644
--- a/src/stalld.c
+++ b/src/stalld.c

